apps/sv/src/test/scala/org/lfdecentralizedtrust/splice/store/db/SvDsoStoreTest.scala

```diff
@@ -667,11 +667,14 @@ abstract class SvDsoStoreTest extends StoreTest with HasExecutionContext {
       val app2Closed = (1 to 3).map(_ => appRewardCoupon(round = 2, userParty(2)))
 
       val sv1NotClosed =
-        (1 to 3).map(_ => svRewardCoupon(round = 3, userParty(1), userParty(1), 1000))
+        (1 to 3).map(_ => svRewardCoupon(round = 3, userParty(4), userParty(1), 1000))
```

Comment:
do we have a test where both beneficiary and SV match? It should work fine but seems like a case worth testing and it's not completely obvious to me that we are atm.